Controllers/CureController.cs

```diff
@@ -97,13 +93,19 @@ public IActionResult RegistrationForm()
                 .First()
             };
 
-            registration.TicketTypes =
-            [
-              new SelectListItem("Gold", "Gold"),
-              new SelectListItem("Silver", "Silver"),
-              new SelectListItem("Sweet", "Sweet"),
-              new SelectListItem("Standard", "Standard"),
-            ];
+            registration.TicketTypes = new List<SelectListItem>();
+            
+            foreach(TicketInventory ti in ticketInventory)
+            {
+                SelectListItem item = new SelectListItem(ti.Name, ti.Name);
+                if (ti.QuantityAvailable == 0)
+                {                 
+                    item.Disabled = true;
+                    item.Text = ti.Name + " - SOLD OUT";
+
+                    registration.TicketTypes.Add(item);
+                }
```

Comment:
This line needs to be outside of the if statement I believe. Right now it's only adding the ticket to the dropdown _if_ it is sold out

```suggestion 
                }
                registration.TicketTypes.Add(item);
```